diagnostic.md

```diff
@@ -5,74 +5,82 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+The purpose of a backend is to store data and persist data.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+The model is used by the controller to fetch data.
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
+The controller communicates with the model.
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+We aren't using views because we are focusing on SPAs--beyond scope of this class.
 ```
 
 What does C.R.U.D stand for?
 
 ```md
-// your response here
+Create, Read, Update, Destroy
 ```
 
 In which part of the MVC pattern can we find C.R.U.D actions?
 
 ```md
-// your response here
+Router
 ```
 
 List at least 5 standard rails actions that C.R.U.D requests correspond to?
 
 ```md
-// your response here
+Index, Show, Create, Update, Destroy
 ```
 
 A user action fires a `GET` request for `/people/1`. Explain in detail each step
 required for data to be returned to the client. (bullet points or ordered list)
 
 ```md
-// your response here
+ran out of time to write more elaborate answer
+~design curl request - use server to send req
+~update routes.rb - router takes a look at where to look and what to do (GET/POST)
+~update controller_file.rb - controller translates the Rails command to SQL and sends to model/DB
+~use localhost to view changes
```

Comment:
The important steps are:

Request hits the router
Router chooses the controller
Controller tells the model what to do
Model gathers data from the database
Model sends data to controller
Controller serves data to the client